### DISCUSSION TITLE
Laravel 10 compatibility

### DISCUSSION THREAD

**Discussion by fusigabs** [binaryk/laravel-restify#548] [Q&A]
It appears this is not yet compatible with Laravel 10 and PHP 8? I get the error:
```
 Problem 1
    - binaryk/laravel-restify[1.0.0, ..., 1.1.4, 2.0.0, ..., 2.3.0] require php ^7.1 -> your php version (8.2.2) does not satisfy that requirement.

```
Any plans to upgrade to L10?

**Comment by ionut-tanasa** [comment]
there's a branch already addressing that, but unsure when's going to be released or if it's ready
